ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -171,6 +172,13 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
     </div>;
   }
 
+  async updateDisabledStatusForUsers(usernameList: string[], disable: boolean) {
+    await authDomainApi().updateDisabledStatusForUsers(
+        {usernameList: usernameList, disabled: disable}).then(_ => {
+    });
+    this.setState({loadingData: false});
+  }
```

Comment:
Does this page currently show disabled state? if so, we may want to update the UI here.